### PR TITLE
Added popup models to the portfolio project

### PR DESCRIPTION
**Pull request by Abass-rashid**
in this PR  I have added a popup model to  the portfolio project using javascript  for both mobile  and  desktop version

### CODE REVIEW

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Abass-rashid  and @tarek102 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 1]
```diff
@@ -60,107 +61,51 @@ <h2><a href="#">LET’S CONNECT</a></h2>
...
             </ul>
-            <button>see project</button>
+            <div class="divider"></div>
+            <div class="action-container">
+                <button class="action-btn see-live" type="button">See Live
+                    <img src="images/github.png" alt="">
+                </button><button class="action-btn see-source" type="button">See Source
+                    <img src="images/github.png" alt="">
+                </button>
             </div>
```
[the remark this answers is not in the document]
Comment (danmainah) [reply index.html:103]: - [ ] @Team great work implementing the changes requested, but your background doesn't have the blur like the Figma design.

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image | image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Team, 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @Abass-rashid and @tarek102 👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉.
> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ Descriptive commit messages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**